config/detekt.yml

```
@@ -186,24 +282,44 @@ naming:
     minimumFunctionNameLength: 3
   FunctionNaming:
     active: true
-    functionPattern: '^([a-z$][a-zA-Z$0-9]*)|(`.*`)$'
+    functionPattern: '([a-z][a-zA-Z0-9]*)|(`.*`)'
```

Consider updating to include support for Compose (i.e. allow capitals for the first character): https://detekt.dev/compose.html#configurations
```suggestion
    functionPattern: '[A-Za-z][a-zA-Z0-9]*)|(`.*`)'
```

---

I think we good here because we have `ignoreAnnotated: ['Composable']`

---

The docs say that `ignoreAnnotated` is for the annotation `@Composable`. Whereas the `functionPattern` is so that the functions that start with a capital won't be mark as a lint error.